attrs_strict/_type_validation.py

```diff
@@ -19,12 +19,14 @@
 try:
     from inspect import signature
 except ImportError:
-    from funcsigs import signature
+    # silencing type error so mypy doesn't complain about duplicate import
+    from funcsigs import signature  # type: ignore
 
 try:
     from itertools import zip_longest
 except ImportError:
-    from itertools import izip_longest as zip_longest
+    # silencing type error so mypy doesn't complain about duplicate import
```

Comment:
this feels unneeded verbose; this would suffice in one-line 

```
# type: ignore # duplicate import
```

Consider though the alternative way of expressing this more correctly:
```
if sys.version_info >= (3,8):
  import
else:
  import
```